.gitmodules

```diff
@@ -1,19 +1,22 @@
-[submodule "libs/include/mavlink/v2.0"]
+[submodule "c_library_v2"]
 	path = third_party/mavlink/include/mavlink/v2.0
 	url = https://github.com/mavlink/c_library_v2.git
-[submodule "libs/gtest"]
+[submodule "gtest"]
 	path = third_party/gtest
 	url = https://github.com/google/googletest
 [submodule "curl-android-ios"]
 	path = core/third_party/curl-android-ios
 	url = https://github.com/dronecore/curl-android-ios.git
-[submodule "libs/tinyxml2"]
+[submodule "tinyxml2"]
 	path = core/third_party/tinyxml2
 	url = https://github.com/leethomason/tinyxml2.git
-[submodule "grpc/server/proto"]
+[submodule "DronCore-Proto"]
```

Comment:
`dronecore-proto`, to be consistend with the others that are lowercase.

Reply:
OK!